packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -611,32 +639,34 @@ export function TransferPanel() {
         }
       }
 
+      // logic: finally, call the transfer function
       let depositForBurnTx
+
       try {
         if (isSmartContractWallet) {
           showDelayedSCTxRequest()
         }
-        depositForBurnTx = await depositForBurn({
+        const transfer = await cctpTransferStarter.transfer({
```

Comment:
nit:
```suggestion
        const { sourceChainTransaction: depositForBurnTx } = await cctpTransferStarter.transfer({
```

Reply:
We cannot do this because `depositForBurnTx` is already declared in L638. We require it before the try block so that we can use it to perform necessary ops later on, if it's undefined.